.ci/build.sh

```diff
@@ -5,7 +5,15 @@ SCRIPT_DIR="$(dirname "$0")"
 set -exuo pipefail
 source "$SCRIPT_DIR/common"
 
-docker build -t $CI_IMAGE_URI --build-arg IMAGE_REPOSITORY=${IMAGE} --build-arg IMAGE_TAG=${COMMIT} "$SCRIPT_DIR/.."
+OUTFILE="$(mktemp -d)/image.tar"
+
+docker buildx build -o type=oci,dest=${OUTFILE} \
+  -t $CI_IMAGE_URI \
+  --platform=${CI_IMAGE_PLATFORMS} \
+  --build-arg "IMAGE_REPOSITORY=${IMAGE}" \
+  --build-arg "IMAGE_TAG=${COMMIT}" "$SCRIPT_DIR/.."
+
+docker load -i "${OUTFILE}" && rm -f ${OUTFILE}
```

Comment:
need to ensure the load of the image to kind if also working

Reply:
Docker load is not working in GitHub Actions